ui/hooks/src/use-dismissed-card.ts

```diff
@@ -1,19 +1,28 @@
 import { useState } from 'react';
-import { storage } from './utils/storage';
 
 const INITIAL_CLOSE_STATE = [];
 const LOCAL_STORAGE_KEY = 'dismissed-card-ids';
 
-const writeToLocalStorage = (key: string, value: string[]): void => {
+interface IStorage {
+  setItem(key: string, value: string): void;
+  getItem(key: string): string;
+  removeItem(key: string): void;
+}
+
+const writeToLocalStorage = (storage: IStorage, key: string, value: string[]): void => {
   if (storage) {
     storage.setItem(key, JSON.stringify(value));
   }
 };
 
-export function useDismissedCard(): [string[], (newClosedValue: string) => void] {
+export function useDismissedCard(
+  mockStorage?: IStorage,
```

Comment:
```suggestion
  storage?: IStorage,
```
The name `mockStorage` makes sense in the context of testing not here :)

Reply:
:) I changed it to draftStorage.

Reply:
I suggest `statusStorage` rather than draftStorage as your intent is saving status.